advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,591 @@
+P00   INFO: execute non-exclusive pg_start_backup(): backup begins after the requested immediate checkpoint completes
+P00   INFO: backup start archive = 00000001000000000000000C, lsn = 0/C000060
+P00   INFO: full backup size = 50.4MB
+P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
+P00   INFO: backup stop archive = 00000001000000000000000C, lsn = 0/C000138
+P00   INFO: check archive for segment(s) 00000001000000000000000C:00000001000000000000000C
+P00   INFO: new backup label = 20210419-142212F
+P00   INFO: backup command end: completed successfully
+
+$ pgbackrest backup --stanza=demo --type=full --repo=2
+P00   INFO: backup command begin ...
+P00   INFO: execute non-exclusive pg_start_backup(): backup begins after the requested immediate checkpoint completes
+P00   INFO: backup start archive = 00000001000000000000000E, lsn = 0/E000028
+P00   INFO: full backup size = 50.5MB
+P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
+P00   INFO: backup stop archive = 00000001000000000000000E, lsn = 0/E000138
+P00   INFO: check archive for segment(s) 00000001000000000000000E:00000001000000000000000E
+P00   INFO: new backup label = 20210419-142414F
+P00   INFO: backup command end: completed successfully
+
+$ pgbackrest backup --stanza=demo --type=incr --repo=1
+P00   INFO: backup command begin ...
+P00   INFO: last backup label = 20210419-142212F, version = ...
+P00   INFO: execute non-exclusive pg_start_backup(): backup begins after the requested immediate checkpoint completes
+P00   INFO: backup start archive = 000000010000000000000010, lsn = 0/10000028
+P00   INFO: incr backup size = 50.5MB
+P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
+P00   INFO: backup stop archive = 000000010000000000000010, lsn = 0/10000100
+P00   INFO: check archive for segment(s) 000000010000000000000010:000000010000000000000010
+P00   INFO: new backup label = 20210419-142212F_20210419-142502I
+P00   INFO: backup command end: completed successfully
+
+$ pgbackrest backup --stanza=demo --type=incr --repo=2
+P00   INFO: backup command begin ...
+P00   INFO: last backup label = 20210419-142414F, version = ...
+P00   INFO: execute non-exclusive pg_start_backup(): backup begins after the requested immediate checkpoint completes
+P00   INFO: backup start archive = 000000010000000000000012, lsn = 0/12000028
+P00   INFO: incr backup size = 50.6MB
+P00   INFO: execute non-exclusive pg_stop_backup() and wait for all WAL segments to archive
+P00   INFO: backup stop archive = 000000010000000000000012, lsn = 0/12000100
+P00   INFO: check archive for segment(s) 000000010000000000000012:000000010000000000000012
+P00   INFO: new backup label = 20210419-142414F_20210419-142556I
+P00   INFO: backup command end: completed successfully
+```
+
+Here, we alternated `full` and `incr` backups in each repository.
```

Comment:
I would add a sentence or two as to why this is interesting.

Reply:
Here it wasn't really that much interesting. It's just an habit I have to illustrate various backup types. The only real interest here would be to show that when the full is expired, the incr is expired too. But that's not a specific behavior of the multi-repo feature, so not really relevant.